e2e/test/scripts/infrastructure-linode/0.1.0/cluster-template-clusterclass-kubeadm.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
CAPL now has a release so we should no longer need to include these files in the repo, they can be pulled by `clusterctl` directly from the latest CAPL release